tome/discord/views.py

```
@@ -241,6 +257,39 @@ def drop_user(request):
+    edit token.
+    """
+
+    body = json.loads(request.body.decode("utf-8"))
+    duid: str = body.get("discord_user_id")
+
+    logger.info(f"Edit decklist request received from {duid}")
+
+    try:
+        participant = Participants.objects.filter(discord_user_id=duid).get()
+    except Participants.DoesNotExist:
+        logger.error("Participant is not currently linked.")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)